modules/private/internal.go

```diff
@@ -17,6 +18,9 @@ import (
 )
 
 func newRequest(ctx context.Context, url, method string) *httplib.Request {
+	if setting.InternalToken == "" {
+		log.Fatal("no internal token, can not send internal request to server. please use correct app.ini")
```

Comment:
```suggestion
		log.Fatal(`The INTERNAL_TOKEN setting is missing from the configuration file: %q.
Ensure you are running in the correct environment or set the correct configuration file with -c.`, setting.CustomConf)
```

I know technically this might expose internal configuration to users but I think that things in this situation are so FUBARred that hiding the configuration file that the program is using is just going to make debugging the problem even more difficult.

Reply:
I think that's fine. Since we all know that there are files like `/etc/shadow` and `/etc/nginx/nginx.conf`.